docs/README.md

```diff
@@ -26,9 +26,17 @@ Earthly's key features/benefits are:
   * **♻️ Reuse, Don't Repeat**  
     Never have to write the same code in multiple builds again. With Earthly, you can reuse targets, artifacts, and images across multiple Earthfiles, even ones in other repositories, in a single line. Earthly is cache-aware, based on the individual hashes of each file, and has shared caching capabilities. So you can create a vast and efficient build hierarchy that only executes the minimum required steps.
 
+## Earthly Cloud
+
+Earthly Cloud is a cloud-based build automation platform that allows you to run your Earthly builds in the cloud, and is compatible with any CI. Earthly Cloud gives teams repeatable pipelines that run exactly the same in CI as on your laptop; has an automatic and instantly available build cache that makes builds faster; and is super simple to use.
+
+Earthly is better logged in as Earthly Cloud includes a generous free tier, including additional capabilities, like sharing cache with your team, remote build runners, shared logs, and shared secrets. To get started, visit the [Earthly Cloud sign up](https://cloud.earthly.dev/login) page.
```

Comment:
avoid `logged in as` which makes it sound like you are logging in as a different user.

```suggestion
Earthly is better logged in since Earthly Cloud includes a generous free tier, including additional capabilities, like sharing cache with your team, remote build runners, shared logs, and shared secrets. To get started, visit the [Earthly Cloud sign up](https://cloud.earthly.dev/login) page.
```

Reply:
Reworded based on Gavin's suggestion.